lib/ruby_lsp/document.rb

```diff
@@ -213,10 +221,19 @@ def locate(node, char_position, node_types: [])
       NodeContext.new(closest, parent, nesting_nodes, call_node)
     end
 
-    sig { returns(T::Boolean) }
-    def sorbet_sigil_is_true_or_higher
-      parse_result.magic_comments.any? do |comment|
-        comment.key == "typed" && ["true", "strict", "strong"].include?(comment.value)
+    sig { returns(SorbetLevel) }
+    def sorbet_level
+      sigil = parse_result.magic_comments.find do |comment|
+        comment.key == "typed"
+      end&.value
+
+      case sigil
+      when "true"
+        SorbetLevel::Enabled
+      when "strict", "strong"
```

Comment:
Oh I didn't know there's even a `strong` level 😮

Reply:
It's quite challenging to use. It will throw error if there are any `T.untyped` bindings for the entire file. It's essentially a full typed mode, where escape hatches are not available.